bpf/flow.h

```diff
@@ -10,6 +10,8 @@ typedef __u16 u16;
 typedef __u32 u32;
 typedef __u64 u64;
```

Comment:
The convention for most BPF code is to use the underscored values instead of type-def'ing like this. Is there any particular reason why you went with these typedefs?

Reply:
no real reason that was the style used in that file and I just kept it as is